styles.css

```diff
@@ -792,3 +796,35 @@
     vertical-align: middle;
     margin: 0 9px 0 0;
 }
+
+.path-mod-studentquiz .createnewquestion {
+    margin: 0.5rem 0;
+}
```

Comment:
In Moodle now, the preferred approach is to use Boostrap classes, rather than custom CSS, if possible, which itis here. https://getbootstrap.com/docs/4.0/utilities/spacing/